### PR TITLE
Adding support for minute schedule

### PR DESCRIPTION
**Pull request by Bpoe**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/schedule.json [hunk 1]
```diff
@@ -418,11 +418,18 @@
...
+        "Month",
+        "Minute"
       ],
       "x-ms-enum": {
         "name": "ScheduleFrequency",
-        "modelAsString": true
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Minute",
```
Comment (jhendrixMSFT) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/schedule.json]: This is broken and the cause of the build failures.  The values listed here must match the above enum values, see https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-enum

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]

File: specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/schedule.json (hunk 1 above)
Comment (Bpoe) [reply specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/schedule.json]: Added all missing enum values

File: specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/schedule.json [hunk 2]
```diff
@@ -418,11 +418,38 @@
...
+        "Minute"
       ],
       "x-ms-enum": {
         "name": "ScheduleFrequency",
-        "modelAsString": true
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "OneTime",
+            "description": ""
```
Comment (jhendrixMSFT) [inline specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/schedule.json]: I think you can omit the empty description elements.

Comment (Bpoe) [reply specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/schedule.json]: Removed

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**